CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Changelog
 
-## 4.11
+## 4.11.3
```

Comment:
Previously the 4.11.{i} has been another PR in the list of 4.11.0 - are we adding another section for these versions?